docs/user-event/api-convenience.mdx

```diff
@@ -18,6 +18,20 @@ click(element: Element): Promise<void>
 pointer([{target: element}, {keys: '[MouseLeft]', target: element}])
 ```
 
+```js
+test('click', async () => {
+  const onChange = jest.fn()
+  render(<input type="checkbox" onChange={onChange} />)
+
+  const checkbox = screen.getByRole('checkbox')
+
+  await userEvent.click(checkbox)
+
+  expect(onChange).toHaveBeenCalledTimes(1)
+  expect(checkbox).toBeChecked()
+})
```

Comment:
Instead of using `userEvent` directly, we should set it up first and create a `user`.
This was a change between v13 and v14, and these examples should also be updated with it.

```suggestion
test('click', async () => {
  const user = userEvent.setup()
  const onChange = jest.fn()
  render(<input type="checkbox" onChange={onChange} />)

  const checkbox = screen.getByRole('checkbox')

  await user.click(checkbox)

  expect(onChange).toHaveBeenCalledTimes(1)
  expect(checkbox).toBeChecked()
})
```

Reply:
@timdeschryver 
Thank you for your review. I modified the example code as you guided me.

https://testing-library.com/docs/user-event/utility
I initially referenced that page, and it's written as if it uses "userEvent" directly.
May I create a Pull Request after making modifications to this page?